user-interface/src/main/java/life/qbic/datamanager/views/projects/project/access/UserAvatarWithNameComponent.java

```diff
@@ -0,0 +1,19 @@
+package life.qbic.datamanager.views.projects.project.access;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import life.qbic.datamanager.views.account.UserAvatar;
+
+/**
+ * A component displaying a user avatar and user name
+ */
+public class UserAvatarWithNameComponent extends Div {
+
+  public UserAvatarWithNameComponent(UserAvatar userAvatar, String userName) {
+    addClassName("avatar-with-name");
```

Comment:
This assigns the same css name as the div containing the avatar and username in the userprofilecomponent which can lead to confusion. I recommend to adapt the css name to avoid confusion.
(Especially since there is currently no styling associated with this component). 

NIT: I'd sanity check the design with SP but IMHO a gap between the avatar and the user name and align the user name in the middle of the avatar would work wonders:

Before: 
![Screenshot 2024-06-19 at 13 34 46](https://github.com/qbicsoftware/data-manager-app/assets/29627977/9d348684-fd39-4905-a6f4-5ea414b5942f)
After:
![Screenshot 2024-06-19 at 13 43 17](https://github.com/qbicsoftware/data-manager-app/assets/29627977/443d1d48-c1a1-409f-8d1c-954e1fdd98cf)

For this the following css rule can be employed: 

`your-awesome-new-css-name {
	display: inline-flex;
	justify-content: center;
	align-items: center;
	column-gap: var(--lumo-space-s);
} `